modules/claims_api/app/sidekiq/claims_api/service_base.rb

```diff
@@ -40,6 +40,11 @@ def retry_limits_for_notification
 
     protected
 
+    def set_state_for_submission(submission, state)
+      submission.status = state
+      submission.save!
+    end
+
```

Comment:
Going back over this code I wrote I realized it was very unnecessarily tied to AutoEstablishedClaims (I mean that is what we were building at the time) so I suspect that some of this could be refactored to be more flexible going forward. 

So even though this method is kind of duplicating already in place logic for Claims, I did not want to start refactoring as part of this because I worried that would spiral out, and it is kind of outside the scope of this, but looking at that as possible clean up tickets.